mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/user/CassandraSubscriptionMapperTest.java

```diff
@@ -38,4 +39,9 @@ protected SubscriptionMapper createSubscriptionMapper() {
     public void tearDown() {
         CLUSTER.clearAllTables();
     }
+
+    @AfterClass
+    public static void stop() {
+        CLUSTER.close();
```

Comment:
s/CLUSTER/cassandra/g

Someone once told me a cluster is something really different ;-)